docs/reference/data-sources/push.md

```diff
@@ -44,14 +44,16 @@ fv = FeatureView(
 ```
 
 ### Pushing data
+Note that the `to` parameter is optional and defaults to online but we can specify these options: PushMode.ONLINE, PushMode.OFFLINE, or PushMode.ONLINE_AND_OFFLINE.
```

Comment:
nit: would wrap these options in `` to make them look nicer

Reply:
done.